### PR TITLE
Add new IO landing page and update builders

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?

Update and add new builders from:

- `react@2.x`
- `pages@0.x`

to:

- `react@3.x`
- `store@0.x`
- `styles@1.x`
- `messages@1.x`

Also, overall improvements such as adding ESLint and Prettier configuration, and adding the new IO landing page from https://github.com/vtex-apps/io-landing-page 

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.
- [x] Refactor

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vtex.landing-pages",
-  "version": null,
+  "version": "0.2.2",
```
Comment (salesfelipe) [inline react/package.json]: no need for this version

File: react/components/IOLanding/Evolution.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
```
Comment (salesfelipe) [inline react/components/IOLanding/Evolution.tsx]: ;?

File: react/components/IOLanding/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from "react";
+import { FormattedMessage } from "react-intl";
+
```
Comment (salesfelipe) [inline react/components/IOLanding/Hero.tsx]: remove empty line

File: react/components/IOLanding/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      </div>
+      <a
+        className="tc c-muted-1 mt5 t-small"
+        href="https://www.vtex.com/partner/program/"
+      >
+        <FormattedMessage id="hero.partners" />
+      </a>
+    </section>
+  );
+};
```
Comment (salesfelipe) [inline react/components/IOLanding/Hero.tsx]: `;`?

File: react/components/IOLanding/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+            </div>
+            <div className="bg-rebel-pink pa6">
+              {intl.formatMessage({ id: "navbar.build" })}
+            </div>
+          </li>
+        </div>
+      </ul>
+    </nav>
+  );
+};
```
Comment (salesfelipe) [inline react/components/IOLanding/Navbar.tsx]: ;?

File: react/components/IOLanding/Navbar.tsx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+      <ul
+        className="flex flex-column flex-row-m justify-between list mt0 c-muted-1 items-center bg-base mb0 relative"
+        style={{
+          boxShadow: `0px 3px 20px rgba(0, 0, 0, 0.3)`
+        }}
+      >
+        <div className="flex items-center">
+          <img src={logoPath} className="h3" alt="VTEX" />
+          <p>|</p>
+          <p className="ml3">Developer</p>
```
Comment (salesfelipe) [inline react/components/IOLanding/Navbar.tsx]: Developer?

File: react/components/IOLanding/Product.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from "react";
+import { FormattedMessage } from "react-intl";
+
+import productImagePath from "../../images/Product.svg";
+
+const ProductDisplay: FunctionComponent = () => {
+  return (
+    <section className="w-100 flex justify-between pv10 bg-base">
+      <div className="flex flex-column justify-center w-33 pl9">
```
Comment (salesfelipe) [inline react/components/IOLanding/Product.tsx]: Why don't add the `c-muted-1` at this div?

File: react/components/IOLanding/StartBuilding.tsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+          <Button variation="secondary">
+            <p className="action-secondary">
+              <FormattedMessage id="startbuilding.cta" />
+            </p>
+          </Button>
+        </div>
+      </div>
+    </div>
+  </section>
+);
```
Comment (salesfelipe) [inline react/components/IOLanding/StartBuilding.tsx]: ;?

File: react/Dreamstore.tsx [hunk 9]
```diff
@@ -9,29 +9,52 @@ export default class Dreamstore extends Component {
       <main>
         <Helmet>
           <title>VTEX Dreamstore - The next generation store framework</title>
```
Comment (salesfelipe) [inline react/Dreamstore.tsx]: intl?

Comment (victorhmp) [reply react/Dreamstore.tsx]: I'll create another PR with adjustments specific to this Dreamstore landing.
I didn't change anything in it, just linting and a broken import

File: react/components/IOLanding/Evolution.tsx (hunk 2 above)
Comment (victorhmp) [reply react/components/IOLanding/Evolution.tsx]: Ooops, these components were added before I added Prettir and ESLint configuration hahaha
Fixed it

File: react/components/IOLanding/Hero.tsx (hunk 4 above)
Comment (victorhmp) [reply react/components/IOLanding/Hero.tsx]: FIxed

File: react/components/IOLanding/Navbar.tsx (hunk 5 above)
Comment (victorhmp) [reply react/components/IOLanding/Navbar.tsx]: Fixed

File: react/components/IOLanding/Navbar.tsx (hunk 6 above)
Comment (victorhmp) [reply react/components/IOLanding/Navbar.tsx]: Yep, this should not change with locale settings, as it basically is "VTEX | Developer"

File: react/components/IOLanding/StartBuilding.tsx (hunk 8 above)
Comment (victorhmp) [reply react/components/IOLanding/StartBuilding.tsx]: Fixed

File: react/components/IOLanding/Hero.tsx (hunk 3 above)
Comment (victorhmp) [reply react/components/IOLanding/Hero.tsx]: Fixed it

File: react/components/IOLanding/Product.tsx (hunk 7 above)
Comment (victorhmp) [reply react/components/IOLanding/Product.tsx]: Sure! Did not notice that! hahaha
Thanks ;)

**Pull request merged**